patches/third_party/ffmpeg/libavutil-tx_template.c.patch

```diff
@@ -0,0 +1,79 @@
+diff --git a/libavutil/tx_template.c b/libavutil/tx_template.c
+index 8dc3d2519c13abec090b374fa5ed0f11f7153849..a5d156d2789ec521c034edd929e04b2ad09a833b 100644
+--- a/libavutil/tx_template.c
++++ b/libavutil/tx_template.c
+@@ -27,6 +27,29 @@
+ #define TABLE_DEF(name, size) \
+     DECLARE_ALIGNED(32, TXSample, TX_TAB(ff_tx_tab_ ##name))[size]
+ 
++#ifdef _WIN32
++
++#include <malloc.h>
++
++// Declares aligned ff_tx_tab_* pointer.
++#define TABLE_DEF_PTR(name, size) \
++  DECLARE_ALIGNED(32, TXSample*, TX_TAB(ff_tx_tab_##name))
++
++// Allocates aligned memory for ff_tx_tab_* variable.
++#define ALLOCATE_FF_TX_TABLE(len)                              \
++  if (!TX_TAB(ff_tx_tab_##len)) {                              \
++    TX_TAB(ff_tx_tab_##len) =                                  \
++        _aligned_malloc((len / 4 + 1) * sizeof(TXSample), 32); \
++  }
++
++#else
++
++// Use static arrays on non-Windows targets as usual.
++#define TABLE_DEF_PTR(name, size) TABLE_DEF(name, size)
++#define ALLOCATE_FF_TX_TABLE(len)
++
++#endif
++
```

Comment:
we cannot use chromium_src for this override, because `tx_template.c` is included into `tx_float.c`.